Zstd support fails for Zstd <= 1.5.0

The function `ZSTD_defaultCLevel` was not part of any release prior to v1.5.0 of Zstd, therefore:

```
/home/dmg/.cpm/bxzstr/d488119fdf94185eb9eaac489709a9e37dbe9c76/include/zstd_stream_wrapper.hpp:41:22: error: use of undeclared identifier 'ZSTD_defaultCLevel'; did you mean 'ZSTD_maxCLevel'?
                        const int level = ZSTD_defaultCLevel(), const int = 0)
```	

Since Ubuntu 22.04 comes with ZStd 1.4.8 this seems like a reasonable thing to try and fix in the meantime

--- a/include/zstd_stream_wrapper.hpp
+++ b/include/zstd_stream_wrapper.hpp
@@ -38,7 +38,7 @@ namespace detail {
 class zstd_stream_wrapper : public stream_wrapper {
   public:
     zstd_stream_wrapper(const bool _isInput = true,
-			const int level = ZSTD_defaultCLevel(), const int = 0)
+			const int level = ZSTD_CLEVEL_DEFAULT, const int = 0)
 	    : isInput(_isInput) {
 	if (this->isInput) {
 	    this->dctx = ZSTD_createDCtx();

